### ISSUE TITLE
Auto Setup: Add script_extension_data.gd

### ISSUE THREAD

**Issue by ithinkandicode** [GodotModding/godot-mod-loader#125]
I think the auto setup process will need to be updated to include the file *script_extension_data.gd*, which as introduced in this PR:

- #124

**Comment by Qubus0** [comment]
right, completely forgot that. should be only this in `mod_loader_setup.gd` though.
```gdscript
{
      "base": "Resource",
      "class": "ScriptExtensionData",
      "language": "GDScript",
      "path": "res://addons/mod_loader/script_extension_data.gd"
}
```

**Comment by ithinkandicode** [comment]
Closed in #127:

- #127
